src/main/java/edu/cornell/kfs/coa/businessobject/CuAccountGlobal.java

```diff
@@ -124,11 +124,40 @@ private Account updateAccountValuesOnAccountGlobalDetailWithNewValuesFromAccount
 			if(ObjectUtils.isNotNull(this.getIndirectCostRecoveryAccounts()) && this.getIndirectCostRecoveryAccounts().size() > 0){
 				updateIcrAccounts(globalDetail, account.getIndirectCostRecoveryAccounts());
 			}
+
+			if (ObjectUtils.isNotNull(this.accountRestrictedStatusCode)) {
+				updateAccountRestrictedStatusCodeForAccountBeingEdited(this.accountRestrictedStatusCode, account);
+			}
 		}
 
 		return account;
 	}
 
+	/**
+	 * The business rules for the accountRestrictedStatusCode is as follows:
+	 * If the sub-fund of the account listed in the edit list of accounts on
+	 * the global maintenance edoc is one that has a default accountRestrictedStatusCode
+	 * listed, then any change entered in the global account maintenance edoc is ignored.
+	 *
+	 * If the sub-fund of the account listed in the edit list of accounts on
+	 * the global maintenance edoc is one that does NOT have a default accountRestrictedStatusCode
+	 * listed, then any change entered in the global account maintenance edoc overwrites
+	 * the accountRestrictedStatusCode value on that particular account from the edit list.
+	 *
+	 * @param globalAccountMaintenanceDocRestrictedStatusCode
+	 * @param accountBeingEdited
+	 */
+	private void updateAccountRestrictedStatusCodeForAccountBeingEdited(String globalAccountMaintenanceDocRestrictedStatusCode, Account accountBeingEdited)
+	{
+		if (ObjectUtils.isNotNull(accountBeingEdited.getSubFundGroup()) && StringUtils.isNotBlank(accountBeingEdited.getSubFundGroup().getAccountRestrictedStatusCode())) {
```

Comment:
I will look at extracting the condition logic into an appropriately named method (even though it may be a on-liner).  I think that for future maintenance it will make the code more readable than having double negatives that others later on will need to sift through.